src/site/content/en/blog/hands-on-portals/index.md

```diff
@@ -18,18 +18,18 @@ A new web platform API called [Portals](https://github.com/WICG/portals) can hel
 <img class="w-screenshot" src="hero.gif" alt="Showing a demo of seamless navigations between two sites using Portals">
 
 ## What Portals enables
-**Portals** is a new HTML element that can help you keep your front-end simple while allowing seamless navigations with custom transitions. Like dynamic navigations in Single Page Applications with an [App Shell model](https://developers.google.com/web/fundamentals/architecture/app-shell),
+Portals are new HTML elements that can help you keep your front-end simple while allowing seamless navigations with custom transitions. Like dynamic navigations in Single Page Applications with an [App Shell model](https://developers.google.com/web/fundamentals/architecture/app-shell),
```

Comment:
"Portals are new HTML elements" kind of sounds like there's more than one type of portal element.

maybe try "Portals are a new HTML element" ?

Reply:
Should be OK to combine plural verb with singular object.

Reply:
Done.